### PR TITLE
add Router and connect pages

### PR DESCRIPTION
**Pull request by pascalweber85**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -19,48 +19,50 @@ export default function App() {
...
   }
 
   function handleClickBack() {
-    setActivePage('LocationPage')
+    push('/')
   }
 
   function toFavorite() {
-    setActivePage('FavoritePage')
+    push('/FavoritePage')
```
Comment (MelGGit) [inline src/App.js:67]: This is just a sugegstion :)
You could implement a Link from react-router-dom in the component where these functions get used to link to the page. like:
`<Link to='/' />`
You would save yourself some code and also some props :)

File: src/pages/DetailsPage.spec.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ describe('DetailsPage', () => {
   it('renders number of headings and an images of the page', () => {
     render(<DetailsPage />)
 
-    // const title = screen.getAllByRole('heading')
-    // expect(title).toHaveLength(4)
+    const title = screen.getAllByRole('heading')
+    expect(title).toHaveLength(4)
```
Comment (MelGGit) [inline src/pages/DetailsPage.spec.js:9]: Im not sure, if you have implemented this test yet. But if so this test could need a bit more.

**Review by MelGGit — APPROVED** [review]
Hey Pascal, well done :) Nice coding. I was able to follow you steps pretty well.
I only had a small suggestion that would save you some code :)
For the test im not sure if you have fully implemented the test yet, since you dont give the component in the test any prop.

File: src/App.js (hunk 1 above)
Comment (pascalweber85) [reply src/App.js:67]: hey thanks Mel , yes i wanna do this in the next step, when i implement a navigation to my app

**Pull request merged**